### PR TITLE
Align scheduler recurrence icon in month view

### LINKED ISSUE #2335: Scheduler icons are misaligned in Month View
**Describe the bug**
Scheduler icons are misaligned in Month View:

- [x] ellipsis icons go too high up (invisible behind events in Default, almost invisible in Bootstrap, invisible in Material) **Not replicable anymore**
- [ ] recurrence icons are not vertically aligned and are partially hidden in **bootstrap**
- [ ]  cell text with the date is too far down and kind of overlaps the events **Logged in the Blazer repo**(https://github.com/telerik/blazor/issues/4120)
- [x] More events button is invisible in Material (its background is white until hovered which matches the cell of the scheduler) **Not replicable anymore**

Reported in 1510396 for the ellipsis icon, **Public in 1510438** 

**To reproduce**

1. Open https://demos.telerik.com/blazor-ui/scheduler/month-view
1. Review it for glitches with all three themes

**Expected behavior**
Icons are aligned, buttons visible, padding between elements proper

2021-03-10 09_52_01-Window
2021-03-10 09_52_19-Window
2021-03-10 10_04_13-Window

**Screenshots**

**Affected package (please remove the unneeded items)**
* theme-bootstrap
* theme-default
* theme-material

**Affected suites (please remove the unneeded items)**
* Telerik UI for Blazor

**Affected browsers (please remove the unneeded items)**
* All

**Build system information (please remove the unneeded items)**
* Not Applicable

**Additional context**
Add any other context about the bug.

### PR DESCRIPTION
**Pull request by NikoPenev21**
closes #2335

### CODE REVIEW

**Review by TeyaVes — APPROVED** [review]

**Review by Juveniel — DISMISSED** [review]
1. Please review [the PR guidelines wiki](https://github.com/telerik/kendo-themes/wiki/Opening-a-new-issue-or-PR#working-on-an-issue-and-opening-a-new-pr). Assign yourself to the issue, add the appropriate labels, add the main issue to the FE project in the current iteration and also set a milestone.
2. The issue describes several problems within the month view and also in different themes (Bootstrap, Material and Default) which are not addressed in this PR. Address all issues with different commits or split this into multiple work items.
3. The reload icon is not actually centered and causes the text below to be displaced(visible in the generated previews). I would suggest to take a look at the `.k-event-actions` container and its layout styles and start from there.

**Comment by NikoPenev21** [comment]
@Juveniel Thanks for the thorough reply. This is my first PR in kendo-themes.
1. I was not sure which project I have to choose: jQuery1 or Front-end
2. I have updated the issue accordingly: 2 issues are no longer replicable (after consulting with @matanasov):
 - _ellipsis icons go too high up (invisible behind events in Default, almost invisible in Bootstrap, invisible in Material) **Not replicable anymore**_
 And 
 - _More events button is invisible in Material (its background is white until hovered which matches the cell of the scheduler) Not replicable anymore_
And for this:
_cell text with the date is too far down and kind of overlaps the events _
I have logged an issue in Blazor as it is a problem with event positioning with JS.
Maybe I should leave a comment in the issue for these updates as well?
3. Are you referring to

**Comment by Juveniel** [comment]
@NikoPenev21 you can add the issue both in FE and jQuery 1 project for tracking purposes. The main issue can be updated as well with information regarding the scenarios that are no longer reproducible.

As for the text - yes this is the exact change I am referring to.

File: packages/bootstrap/scss/scheduler/_layout.scss [hunk 1]
```diff
@@ -25,6 +25,9 @@
         .k-event-actions {
             right: $padding-x-sm;
             font-size: $font-size-sm;
+            &:first-child {
+                margin: .1ex .4ex 0 $padding-x-sm;
```
Comment (NikoPenev21) [inline packages/bootstrap/scss/scheduler/_layout.scss:29]: After much consideration and comparison to how this [has been styled in the default theme](https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/scheduler/_layout.scss#L461) , the `ex` unit for the top margin is left hard-coded.

**Comment by NikoPenev21** [comment]
@telerik/kendo-theme-reviewers please review

**Review by tsvetomir — APPROVED** [review]
LGTM

**Comment by TeyaVes** [comment]
I was rather wondering whether we can use display: flex for .k-event and align the elements inside without absolute positions, margins, paddings, etc. I'm gonna test it and see whether it'll work

**Comment by TeyaVes** [comment]
It looks like changing the .k-event to display: flex would require more time and changes to accommodate the rendering in the different suites and all of the possible layout cases, so I've opened up a separate issue regarding it: https://github.com/telerik/kendo-themes/issues/3550

For now Niko's solution seems to fix the current issue.

**Review by TeyaVes — APPROVED** [review]

**Comment by Juveniel** [comment]
@NikoPenev21 you can rebase the branch from develop in order to remove the incorrectly generated barcode images.

**Review by Juveniel — APPROVED** [review]

**Review by joneff — APPROVED** [review]
The solution still looks iffy to me, but should do for now.

**Pull request merged**